src/auto_contiguous.cpp

```diff
@@ -64,7 +64,9 @@ void auto_contiguous::apply(module& m) const
         if(ins->outputs().empty() and ins != last)
             continue;
         shape s = ins->get_shape();
-        if(not s.dynamic() and not s.standard() and s.elements() != 0)
+        // If s is not standard layout or has out of sequence strides, insert "contiguous" op
+        // to make a standard shape
+        if(not s.dynamic() and (s.as_standard() != s) and s.elements() > 1)
```

Comment:
Instead of `s.as_standard() != s`, we probably need to make it `not s.standard() or s.normalize_standard() != s` like it was before, that should avoid the `t != shape::tuple_type` assertion failures.

Reply:
That cleared most of the errors, but it's still failing in onnx parsing.  Investigating...